First-Peoples-Cultural-Council/fv-be
# Fw 4434 add management command to rebuild the index with current data

### Description of Changes
Please briefly explain the changes here.

### Checklist
- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- ~Tests have been updated / created if applicable~
- ~Admin Panel has been updated if models have been added or modified~
- ~Migrations have been updated and committed if applicable~

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [x] Pull the branch and test locally

### Additional Notes
To test locally:
1. Ensure that elastic-search server is running locally. Instructions can be found in the readme to get the docker container running.
2. Run the following command from firstvoices folder:
`python3 manage.py rebuild_index`
3. It should delete the existing index, and recreate the index with all the dictionary entry documents present in the database.
4. Verify that the documents are indexed by searching for them. All documents can be fetched if `search_term` is not provided, or you can try randomly search for some entries.
